src/ccache.c

```diff
@@ -3461,6 +3462,7 @@ ccache_main_options(int argc, char *argv[])
 		{"help",          no_argument,       0, 'h'},
 		{"max-files",     required_argument, 0, 'F'},
 		{"max-size",      required_argument, 0, 'M'},
+		{"get-config",    required_argument, 0, 'k'},
```

Comment:
Please keep the option list in sorted order.

Reply:
To clarify, you expect to sort on the first component (`get-config` here), not the last component (`k`), right? But then those will not be in the same order as the `getopt_long` string argument?

Reply:
I sorted on the short component here (the long name, not the short name). Note that the list in master is already not sorted, with `print-config` after `set-config`.